sdk/iOS/src/MSPullSettings.m

```diff
@@ -0,0 +1,37 @@
+// ----------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ----------------------------------------------------------------------------
+
+#import "MSPullSettings.h"
+
+#pragma mark * MSPullSettings Implementation
+
+@implementation MSPullSettings
+
+static NSInteger const MSDefaultPageSize = 50;
+
+#pragma mark * Initializer method(s)
+
+- (instancetype)init {
```

Comment:
I feel like this class needs an initWithPageSize constructor for now, with only 1 property it feels awkward to use in the current form.